week-1-practise/level-1/3-syntax-error-3.js

```diff
@@ -1,6 +1,6 @@
 // There is a syntax error with this code. Fix it.
 
-function(a, b, c) {
+let addNumbers = function (a, b, c) {
     return a + b + c;
 }
 
```

Comment:
With this one, you've created an anonymous function and have assigned it to a variable. You could also create it as a named function which works equally well ass follows:

```
function addNumbers(a, b, c) {
    return a + b + c;
}

var result = addNumbers(1, 3, 4);
console.log(result); // Expected result 8 - Run the program and make sure it works
```